daphne/src/vdaf/test_vec/XofHmacSha256Aes128.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
maybe name this dir `xof_test_data`? `test_vec` doesn't mean anything to me

Reply:
`test_vec` is consistent with the base draft and libprio: https://github.com/divviup/libprio-rs/tree/main/src/vdaf/test_vec